<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph2dVerificationSettingDialog</class>
 <widget class="QDialog" name="Graph2dVerificationSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Verification Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="timestepGroupBox">
     <property name="title">
      <string>Timestep</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="timestepLabel">
        <property name="text">
         <string>Select the timestep of calculation result</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SliderWithValue" name="timestepSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="calculationResultGroupBox">
     <property name="title">
      <string>Calculation Result</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="calculationResultLabel">
        <property name="text">
         <string>Select which physical value to use for comparison.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
        <item row="0" column="0">
         <widget class="QLabel" name="gridLabel">
          <property name="text">
           <string>Grid:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="gridComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="physicalValueLabel">
          <property name="text">
           <string>Physical Value:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="physicalValueComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="calculationResultGroupBox_2">
     <property name="title">
      <string>Measured Values</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="measuredValueLabel_2">
        <property name="text">
         <string>Select which measured value to use for comparison.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
        <item row="0" column="0">
         <widget class="QLabel" name="fileLabel">
          <property name="text">
           <string>File:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="fileComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="measuredValueLabel">
          <property name="text">
           <string>Value:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="measuredValueComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderWithValue</class>
   <extends>QSlider</extends>
   <header>guibase/widget/sliderwithvalue.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Graph2dVerificationSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Graph2dVerificationSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
